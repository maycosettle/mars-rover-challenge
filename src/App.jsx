import { Photos } from 'pages';

const App = () => <Photos />;

export default App;
