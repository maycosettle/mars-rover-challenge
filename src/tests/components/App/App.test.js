import { shallow } from 'enzyme';
import App from 'App';

describe('tests on <App />', () => {
  const wrapper = shallow(<App />);

  test('Should render App correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
