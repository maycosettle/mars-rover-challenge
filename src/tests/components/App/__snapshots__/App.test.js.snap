// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests on <App /> Should render App correctly 1`] = `<Photos />`;
